database/migrations/20221114220259-create-wallet.js

```diff
@@ -17,6 +17,10 @@ module.exports = {
           key: 'id',
         },
       },
+      account_number: {
```

Comment:
Shall we consider making account number have unique constraint??

Reply:
yes. It should be unique